en/Advanced topics/Commercial license.md

```diff
@@ -1,53 +1,53 @@
 ### About Obsidian commercial license
 
-Obsidian is free for personal use. For commercial use in a company of at least 2 people, you need to purchase a commercial license in order to use it for work-related activities such as taking meeting notes or keeping track of a team project.
+Obsidian is free for personal use. For commercial (business) use in a company of two (2) or more employees, a commercial license is required. Business use includes any work-related activities such as capturing meeting notes, maintaining team project notes, configuring an organizational wiki, and similar note taking / document management / knowledge management functionality.
 
-To purchase commercial licenses, please go to [our pricing page](https://obsidian.md/pricing).
+To purchase a commercial license, please visit [our pricing page](https://obsidian.md/pricing).
 
 ### Frequently asked questions
 
 ##### Can I try Obsidian for commercial use before paying?
 
-Before you commit to Obsidian though, you can evaluate it for commercial use for 14 days.
+Yes. You may evaluate Obsidian for commercial use for 14 days.
 
-##### Do non-profit organizations need commercial license?
+##### Do non-profit organizations require a commercial license?
 
-No, non-profit organizations can use Obsidian without commercial licenses.
+No, non-profit organizations may use Obsidian without a commercial license.
 
 ##### Are there bulk discounts?
 
 For bulk discount inquiry, please contact us at support@obsidian.md.
 
 ##### How can I ask my company to purchase licenses for me or my team?
 
-You can ask your IT department to reach out to support@obsidian.md if they have questions or need paperwork from us.
+Ask your IT department to contact support@obsidian.md if they have questions or require any specific paperwork.
 
 ##### Do I need a commercial license?
 
 From our EULA:
 
 > If you use OBSIDIAN for commercial use, you must obtain a commercial license. Commercial use is defined as using OBSIDIAN for work-related activities in a company with two (2) or more employees.
 
-Some examples that might help you understand if you need a commercial license in your case:
+Some examples to help determine if you require a commercial license:
 
 > *Q1. I'm a freelancer/writer/blogger, do I need a commercial license?*
 > \
-> A: Unless you hire at least one person full time, you do not need a commercial license.
+> A: Unless you hire at least one additional person, you do not need a commercial license.
 
 > *Q2. I'm a teacher/professor and I use Obsidian for my teaching/lectures, do I need a commercial license?*
 > \
-> A: No, if it's related to education in general, even if you work in a for-profit institution, you do not need a commercial license.
+> A: Obsidian usage for education-related activities within an .edu organization does not require a commercial license. 
```

Comment:
Could you check ".edu organization" to something more generic? Canadian universities don't end with .edu, for example. Maybe schools and educational institution?

Reply:
Good catch. I'll add "schools and other recognized educational institutions."